ecosystem/sep-0006.md

```diff
@@ -301,9 +361,91 @@ const result = await this.server.submitTransaction(transaction)
 
 Predicates are one of the [claimable balance parameters](https://developers.stellar.org/docs/glossary/claimable-balance/#parameters) used to craft a Claimable Balance transaction. They are conditions that must be satisfied in order for the recipient to claim the balance. Anchors are free to set whichever predicates they feel are necessary in order to claim the balance. If there are no predicate preferences, `UNCONDITIONAL` allows accounts to claim balances at anytime.
 
+## Deposit With Indicative Quote
+
+If the anchor supports [SEP-38] quotes, it can provide a deposit that makes a bridge between non-equivalent tokens by receiving, for instance BRL via bank transfer and in return sending the equivalent value (minus fees) as USDC to the user's Stellar account.
+
+The `/deposit_indicative_quote` endpoint allows a wallet to get deposit information from an anchor when the user intends to make a conversion between non-equivalent tokens using an [indicative quote](https://www.investopedia.com/terms/i/indicativequote.asp). With this endpoint, a user has all the information needed to initiate a deposit and it also lets the anchor specify additional information (if desired) that the user must submit via [SEP-12](sep-0012.md).
```

Comment:
Could we use hyphens instead of underscores? There's no hard technical reason for this, but over time most APIs style guides including those at larger tech companies have required hyphens as separators and they're mostly the norm. [Ref](https://www.google.com/search?&q=api+hyphen+or+underscore+in+api)

Reply:
Fixed in https://github.com/stellar/stellar-protocol/pull/973/commits/c40b687c40c81dc2e96b15c95675c1dc7bf1fdd9